pkg/operator/utils/gcp/utils_test.go

```diff
@@ -139,7 +139,7 @@ func TestCheckPermissionsAgainstPermissionListChunking(t *testing.T) {
 
 				chunkCopy := chunk
 
-				mockCall := mockClient.EXPECT().TestIamPermissions("fake", gomock.Any()).DoAndReturn(
+				mockCall := mockClient.EXPECT().TestIamPermissions("fake", gomock.Any()).AnyTimes().DoAndReturn(
```

Comment:
Should these really be `AnyTimes()` or could they be `Times(1)` (just to keep the test case expectations tighter)?

Reply:
This was helpful. It caught some failing tests. This might need more careful review.